config/config_tests.xml

```diff
@@ -273,6 +273,19 @@ NODEFAIL          Tests restart upon detected node failure. Generates fake failu
     <CONTINUE_RUN>FALSE</CONTINUE_RUN>
   </test>
 
+  <test NAME="IRT">
+    <DESC>exact restart from startup, default 6 days + 5 days with restart from interum file</DESC>
+    <INFO_DBUG>1</INFO_DBUG>
+    <STOP_OPTION>ndays</STOP_OPTION>
+    <STOP_N>11</STOP_N>
+    <REST_N>$STOP_N / 2 + 1</REST_N>
```

Comment:
I think this needs to be `$STOP_N / 2 - 1` rather than `+ 1` for this test to work as intended. Have you checked to make sure the test actually creates two restart directories?

Reply:
Actually, just doing `$STOP_N / 2` should be sufficient.

Reply:
I ran into a problem with STOP_N / 2  as the second run is a day (or an option unit) short if you do that.  I have confirmed that STOP_N / 2 + 1 is the correct setting.

Reply:
I'm wrong, update on the way.